velodrome/config.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+import yaml
+import StringIO
+
+CONFIG = "config.yaml"
+
+DEPLOYMENTS = {
+    "fetcher": "fetcher/deployment.yaml",
+    "transform": "transform/deployment.yaml",
+    "influxdb": "grafana-stack/influxdb.yaml",
+    "prometheus": "grafana-stack/prometheus.yaml",
+    "prometheus-config": "grafana-stack/prometheus-config.yaml",
+    "grafana": "grafana-stack/grafana.yaml",
+    "grafana-config": "grafana-stack/grafana-config.yaml",
+    "nginx": "grafana-stack/nginx.yaml",
+    "sqlproxy": "mysql/sqlproxy.yaml",
+}
+
+
+def main():
+    if len(sys.argv) != 1:
+        print >> sys.stderr, "Too many arguments. Specify"
```

Comment:
Looks like I was typing something when a squirrel came in the room ... I don't know?